client/app/scripts/superdesk-templates/templates.js

```diff
@@ -180,12 +183,14 @@
                  * @return {Object} d Returns time object
                  */
                 $scope.getTime = function getTime(time) {
-                    var d = new Date();
+                    if (time) {
+                        var d = new Date();
 
-                    d.setUTCHours(time.substr(0, 2));
-                    d.setUTCMinutes(time.substr(2, 2));
+                        d.setUTCHours(time.substr(0, 2));
+                        d.setUTCMinutes(time.substr(2, 2));
 
-                    return d;
+                        return d;
+                    }
```

Comment:
As of it's not relevant here. We can deal with Timezone across superdesk in a separate ticket. There's plan for this.